graph/tags.go

```diff
@@ -24,8 +24,10 @@ import (
 	"github.com/docker/libtrust"
 )
 
-const DEFAULTTAG = "latest"
+// DefaultTag defines the default tag used when performing images related actions and no tag strings is specified
```

Comment:
"no tag string"

Maybe can be shortened to "DefaultTag is the default string to be used when a tag is required and not specified"